clinvar_ingest/cloud/bigquery/bq-get-schemas.bash

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
```

Comment:
Is this called anywhere in the production pipeline?  If not I'd move to a `bin/misc` folder and comment that it's just for human debugging.

Reply:
This was only used manually to scrape the bigquery schemas created by DSP and write those json files in this directory. Makes sense to put it in a separate folder because we don't need to include this in deployments